SAME_USER_ERROR = "{'username': [ErrorDetail(string='A user with that username already exists.', code='unique')]}"
WORK_IN_PROGRESS = "We're working on this issue"
